coops-ui/src/components/Dashboard/CoaWarningDialog.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent>
+    <v-card>
+      <v-card-title>Address Change Effective 12:01 AM</v-card-title>
+      <v-card-text>
+        <p class="genErr">Address changes take effect at 12:01 AM (PST). No other filings can be performed until an
```

Comment:
As in another comment, this should probably be changed to "Pacific Time" so we don't have to worry about Standard vs Daylight hours.

Reply:
k sounds great, will adjust np.

Reply:
Code fixed, thanks.